### PR TITLE
docs: describe deployment of the minimal image

### PR DESCRIPTION
**Pull request by marquiz**

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
I am not a good reviewer for HELM stuff
/uncc @ArangoGutierrez 
/cc  e0ne

**Comment by mythi** [comment]
LGTM but not tested

File: docs/get-started/deployment-and-usage.md [hunk 1]
```diff
@@ -148,6 +156,13 @@ The command deploys Node Feature Discovery on the Kubernetes cluster in the
 default configuration.  The Configuration section describes how it can be
 configured during installation.
 
+In order to deploy the [minimal](#minimal) image you need to override the image
+tag:
+
+```bash
+helm install node-feature-discovery ./node-feature-discovery/ --set image.tag={{ site.release }}-minimal --namespace $NFD_NS --create-namespace
```
Comment (e0ne) [inline docs/get-started/deployment-and-usage.md:163]: It's not a valid bash command. Where {{site.release}} is  defined?

**Review by e0ne — CHANGES_REQUESTED** [review]

File: docs/get-started/deployment-and-usage.md (hunk 1 above)
Comment (marquiz) [reply docs/get-started/deployment-and-usage.md:163]: It comes from the Jekyll config:
https://github.com/kubernetes-sigs/node-feature-discovery/blob/ea54b7b98a758b9629471bd4abb8294f9943a04f/docs/_config.yml#L49

Comment (e0ne) [reply docs/get-started/deployment-and-usage.md:163]: Thanks for the explanation

**Review by e0ne — APPROVED** [review]

**Review by adrianchiris — COMMENTED** [review]
/lgtm

**Pull request merged**